plugins/module_utils/mh/base.py

```diff
@@ -28,6 +29,17 @@ def __init__(self, module=None):
         if not isinstance(self.module, AnsibleModule):
             self.module = AnsibleModule(**self.module)
 
+        # in 12.0.0 remove this if statement entirely
+        if hasattr(self, 'debug'):
+            self.module.warn(
```

Comment:
@felixfontein I thought it wouldn't make sense to use `deprecate()` here since nothing is really deprecated (well, the vacancy for name `debug` is deprecated, but it feels weird to describe it like that), so I used `warn()` instead. Please let me know if that sits well with you, otherwise how should I express that. TIA

Reply:
I think `deprecate()` would still be better, because a) this has a specific version when something will change, and b) that will make the sanity tests fail if we forget to actually implement this :)

Reply:
fair enough